lib/include/ert/util/test_work_area.hpp

```diff
@@ -38,15 +79,12 @@ extern "C" {
   void                  test_work_area_free(test_work_area_type * work_area);
   const char          * test_work_area_get_cwd( const test_work_area_type * work_area );
   const char          * test_work_area_get_original_cwd( const test_work_area_type * work_area );
-  void                  test_work_area_install_file( test_work_area_type * work_area , const char * input_src_file );
-  void                  test_work_area_copy_directory( test_work_area_type * work_area , const char * input_directory);
-  void                  test_work_area_copy_directory_content( test_work_area_type * work_area , const char * input_directory);
-  void                  test_work_area_copy_file( test_work_area_type * work_area , const char * input_file);
-  bool                  test_work_area_copy_parent_directory( test_work_area_type * work_area , const char * input_path);
-  bool                  test_work_area_copy_parent_content( test_work_area_type * work_area , const char * input_path);
-  void                  test_work_area_sync( test_work_area_type * work_area);
```

Comment:
Appears that you are removing the sync-function from the C-API, the tests and the usage of it, but still exposing it in the `C++`-api?

Reply:
OK - completely removed.